addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -0,0 +1,592 @@
+    'NO': '0190',  #TODO: Not sure I can include it here or if I should override _get_partner_party_vals
+    'HU': 9910,
+    'ES': 9920,
+    'AD': 9922,
+    'AL': 9923,
+    'BA': 9924,
+    'BE': 9925,
+    'BG': 9926,
+    'CH': 9927,
+    'CY': 9928,
+    'CZ': 9929,
+    'DE': 9930,
+    'EE': 9931,
+    'UK': 9932,
+    'GR': 9933,
+    'HR': 9934,
+    'IE': 9935,
+    'LI': 9936,
+    'LT': 9937,
+    'LU': 9938,
+    'LV': 9939,
+    'MC': 9940,
+    'ME': 9941,
+    'MK': 9942,
+    'MT': 9943,
+    'NL': 9944,
+    'PL': 9945,
+    'PT': 9946,
+    'RO': 9947,
+    'RS': 9948,
+    'SI': 9949,
+    'SK': 9950,
+    'SM': 9951,
+    'TR': 9952,
+    'VA': 9953,
+    'SE': 9955,
+    'FR': 9957,
+}
+
+
+class AccountEdiCommon(models.AbstractModel):
+    _name = "account.edi.common"
+    _description = "Common functions for EDI documents: generate the data, the constraints, etc"
+
+    # -------------------------------------------------------------------------
+    # HELPERS
+    # -------------------------------------------------------------------------
+
+    def format_float(self, amount, precision_digits):
+        if amount is None:
+            return None
+        return float_repr(amount, precision_digits)
+
+    def _cleanup_xml_content(self, xml_content):
+        parser = etree.XMLParser(remove_blank_text=True)
+        tree = etree.fromstring(xml_content, parser=parser)
+
+        def cleanup_node(parent_node, node):
+            # Clean children nodes recursively.
+            for child_node in node:
+                cleanup_node(node, child_node)
+
+            # Remove empty node.
+            if parent_node is not None and len(node) == 0 and not (node.text or '').strip():
+                parent_node.remove(node)
+
+        cleanup_node(None, tree)
+
+        return etree.tostring(tree, pretty_print=True, encoding='unicode')
+
+    # -------------------------------------------------------------------------
+    # ELECTRONIC ADDRESS SCHEME (EAS), see https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/
+    # -------------------------------------------------------------------------
+
+    def _get_eas_mapping(self):
+        return COUNTRY_EAS
+
+    # -------------------------------------------------------------------------
+    # UNIT OF MEASURE
+    # -------------------------------------------------------------------------
+
+    def _get_uom_unece_mapping(self):
+        return UOM_TO_UNECE_CODE
+
+    def _get_uom_unece_code(self, line):
+        # list of codes: https://docs.peppol.eu/poacc/billing/3.0/codelist/UNECERec20/
+        # or https://unece.org/fileadmin/DAM/cefact/recommendations/bkup_htm/add2c.htm (sorted by letter)
+        xmlid = line.product_uom_id.get_external_id()
+        if xmlid and line.product_uom_id.id in xmlid and len(xmlid[line.product_uom_id.id].split('.')) == 2:
+            return UOM_TO_UNECE_CODE.get(xmlid[line.product_uom_id.id].split('.')[1], 'C62')
+        return 'C62'
```

Comment:
```suggestion
        external_ids = line.product_uom_id.get_external_id()
        if external_ids:
            _module, xml_id = external_ids[line.product_uom_id].split('.')
        else:
            xml_id = None
        return UOM_TO_UNECE_CODE.get(xml_id, 'C62')
```

Reply:
I adapted the code + I took the change in `UOM_TO_UNECE_CODE` (add the "uom." prefix) into account